zeppelin-interpreter/src/main/java/org/apache/zeppelin/scheduler/Job.java

```diff
@@ -182,22 +182,16 @@ public void run() {
       this.exception = null;
       errorMessage = null;
       dateFinished = new Date();
-      progressUpdator.terminate();
-    } catch (NullPointerException e) {
```

Comment:
Both catch block were identical and `catch (Throwable e)` can handle `NullPointerException` too